python/dask_cudf/dask_cudf/io/tests/test_s3.py

```diff
@@ -19,6 +17,13 @@
 requests = pytest.importorskip("requests")
 s3fs = pytest.importorskip("s3fs")
 
+from moto.server import ThreadedMotoServer  # noqa: E402
```

Comment:
Instead of the noqa, you can put this conditional import right below `moto = pytest.importorskip("moto", minversion="3.1.6")`:
```suggestion
ThreadedMotoServer = pytest.importorskip("moto.server").ThreadedMotoServer
```
However, I suspect flake8 may complain that `moto` is defined but unused. In that case, you may want to skip the assignment and write:
```python
pytest.importorskip("moto", minversion="3.1.6")
ThreadedMotoServer = pytest.importorskip("moto.server").ThreadedMotoServer
```

Reply:
Done, made this change 👍